source/_posts/setup-tailwindcss_parcel_npm.md

```diff
@@ -0,0 +1,237 @@
+* At the beggining it will take time to define your components, but as time progresses you will have a clean css file that are truly components.
+* You're not really using the cascading part of CSS. Some people like it, some people don't. I don't like the cascade, it's like inheritance in OOP without good constraints. Just pick your poison.
+
+## How to set it up?
+
+I will show you how to set it up with two tools: parcel and npm. I think Parcel is the easiest one and I might do a post about Webpack.
+
+__Prerequisites__: 
+
+* Node 
+* npm 
+
+### Parcel
+
+You can copy paste the following lines and paste them into your terminal, and explanation will come afterwards:
+
+```
+mkdir new_tailwind_project && cd $_ \
+&& npm init -y \
+&& npm install -E -D tailwindcss autoprefixer cssnano @fullhuman/postcss-purgecss \
+&& touch postcss.config.js \
+&& ./node_modules/.bin/tailwind init \
+&& mkdir src && touch src/tailwind.css && touch src/index.html \
+&& npm i -g parcel-bundler
+```
+
+Description by line:
+
+1. Create new directory, change `new_tailwind_project` to whaterver name you want.
+1. Create a `package.json` file
+1. Add required development dependencies with explicit version numbers, for example tailwind will be: `0.6.4`
+1. Create an empty postcss configuration file.
+1. Generate tailwind configuration file.
+1. Create `src` folder and the main CSS file where you can define your custom css.
+1. Make sure that parcel is installed.
+
+[Follow common configuration](#common-configuration) and then come back to generate your files.
+
+** HTML **
+
+Because Parcel works by parsing all your files, you have to add the following line to `src/index.html` after `<title>`:
+
+```html
+<!-- ... -->
+<title>Tailwindcss starter</title>
+<!-- add the following line -->
+<link rel="stylesheet" href="./tailwind.css">
+<!-- ... -->
+```
+
+In your terminal type:
+
+```
+>open http://localhost:1234 && parcel src/index.html
+```
+
+Wait for the build in your terminal and the browser should auto-refresh when done. Now you can start modifying the HTML and continue your work.
+
+### npm
+
+You can copy paste the following lines and paste them into your terminal, and explanation will come afterwards:
+
+```
+mkdir new_tailwind_project && cd $_ \
+&& npm init -y \
+&& npm install -E -D postcss-cli tailwindcss autoprefixer cssnano @fullhuman/postcss-purgecss \
+&& touch postcss.config.js \
+&& ./node_modules/.bin/tailwind init \
+&& mkdir src && touch src/tailwind.css && touch src/index.html
+```
+
+Description by line:
+
+1. Create new directory, change `new_tailwind_project` to whaterver name you want.
+1. Create a `package.json` file
+1. Add required development dependencies with explicit version numbers, for example tailwind will be: `0.6.4`
+1. Create an empty postcss configuration file.
+1. Generate tailwind configuration file.
+1. Create `src` folder and the main CSS file where you can define your custom css.
```

Comment:
You have repeated chunks of text, here from line 106 to 124 is repeated previously from lines 63 to 80

Reply:
ohhh, there was a reason why it was repeated